pkg/providers/vsphere/placement/zone_placement.go

```diff
@@ -48,9 +48,9 @@ type Result struct {
 	PoolMoRef                vimtypes.ManagedObjectReference
 	Datastores               []DatastoreResult
 
-	needZonePlacement      bool
-	needHostPlacement      bool
-	needDatastorePlacement bool
+	NeedZonePlacement      bool
```

Comment:
Please add the appropriate GoDocs now that these are exported symbols.

Reply:
Dropped this change as it's not needed anymore.